### PR TITLE
[Bug] [connector-v2] PostgreSQL versions below 9.5 are compatible use cdc sync problem

### PR DESCRIPTION
**Pull request by Lifu12**
https://github.com/apache/seatunnel/issues/5110
…h CDC

## Check list

* [ ] Code changed are covered with tests, or it does not need tests for reason:
* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/seatunnel/tree/dev/docs
* [ ] If you are contributing the connector code, please check that the following files are updated:
  1. Update change log that in connector document. For more details you can refer to [connector-v2](https://github.com/apache/seatunnel/tree/dev/docs/en/connector-v2)
  2. Update [plugin-mapping.properties](https://github.com/apache/seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  3. Update the pom file of [seatunnel-dist](https://github.com/apache/seatunnel/blob/dev/seatunnel-dist/pom.xml)
* [ ] Update the [`release-note`](https://github.com/apache/seatunnel/blob/dev/release-note.md).

### CODE REVIEW

**Comment by TaoZex** [comment]
Thanks for your contribution. It would be better if you changed the title of this pr. Is there any mention of this in the PostgreSQL documentation?

**Comment by Lifu12** [comment]
PostgreSql 9.5 version document https://www.postgresql.org/about/press/presskit95/ru/ mention

**Comment by Hisoka-X** [comment]
Can you add test case for this feature? Thanks. By the way, you can use `mvn spotless:apply` to format your code before commit.

The doc also should metion this change.

**Comment by Hisoka-X** [comment]
cc @ic4y @hailin0

**Comment by TaoZex** [comment]
Please fix code style. Refer : https://github.com/apache/seatunnel/blob/02114db626a290502d33797dfbe9a11ee5b52b6f/docs/en/contribution/coding-guide.md

**Comment by Lifu12** [comment]
update success

**Comment by Lifu12** [comment]
use mvn spotless:apply format code success,
add doc

File: seatunnel-connectors-v2/connector-jdbc/src/test/java/org/apache/seatunnel/connectors/seatunnel/jdbc/internal/dialect/PostgresDialectFactoryTest.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+    @Test
+    public void testPostgresDialectCreate() {
+        PostgresDialectFactory postgresDialectFactory = new PostgresDialectFactory();
+        JdbcDialect postgresLow = postgresDialectFactory.create("postgresLow");
+        String[] fields = {"id", "name", "age"};
+        String[] uniqueKeyField = {"id"};
+        Optional<String> upsertStatement =
+                postgresLow.getUpsertStatement("test", "test_a", fields, uniqueKeyField);
+        System.out.println(upsertStatement);
```
Comment (Hisoka-X) [inline seatunnel-connectors-v2/connector-jdbc/src/test/java/org/apache/seatunnel/connectors/seatunnel/jdbc/internal/dialect/PostgresDialectFactoryTest.java]: Please use `assert`, not only print

Comment (Lifu12) [reply seatunnel-connectors-v2/connector-jdbc/src/test/java/org/apache/seatunnel/connectors/seatunnel/jdbc/internal/dialect/PostgresDialectFactoryTest.java]: ok

**Comment by Lifu12** [comment]
@Hisoka-X

**Review by ic4y — APPROVED** [review]
Nice!

**Review by Hisoka-X — APPROVED** [review]
Let's merge this. But add a e2e test case for older pg version would be better. We can do it later.

**Pull request merged**